tests/integration/test_webhook.py

```diff
@@ -40,3 +40,39 @@ def test_webhook_create_update(project, rand_gen):
         "Topics must be List[Webhook.Topic]. Found `invalid..`"
 
     webhook.delete()
+
+
```

Comment:
These, in my opinion, should be considered unit-tests.

Reply:
There's a lot of tests in `integration` that could be better considered a unit test, but this needs a configured project and client which only exist in integration fixtures, and pulling them out would be too much of a pain for this I think.

Reply:
`client` and `project` could be mocks, the exceptions are thrown before these are used.

Reply:
Good point. I moved them to unit.

Reply:
Second Klaus comments, after that can approve

Reply:
It should be fixed now and checks are passing, can I get an approval?